docs/install/containers.md

```diff
@@ -77,6 +77,22 @@ docker run --rm \
   --output harvest-compose.yml
 ```
 
+By default, the above command uses the harvest configuration file located in the current directory, denoted as `$(pwd)/harvest.yml`. However, you may want to use a harvest config from a different location.
```

Comment:
maybe
By default, the above command uses the harvest configuration file (`harvest.yml`) located in the current directory. If you want to use a harvest config from a different location.